sdk/monitor/monitor-ingestion/review/monitor-ingestion.api.md

```diff
@@ -5,12 +5,22 @@
 ```ts
 
 import { CommonClientOptions } from '@azure/core-client';
+import { OperationOptions } from '@azure/core-client';
 import { TokenCredential } from '@azure/core-auth';
 
+// @public
+export class AggregateUploadLogsError extends Error {
```

Comment:
Where is this used? Do we want to export it?

Reply:
This is the type of error that is thrown at the end